src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 20px;
+  height: 16px;
```

Comment:
will be better to set correct size for the stars (16px - 16px), and set correct margins between them